File: C++/selection_sort.cpp
#include<iostream>

using namespace std;

int main() {
	int N = 5;
	int aux, menor, pos;
	int vetor[N];

	for (int i = 0; i < N; i++) {
		cout << "Digite o " << i + 1 << "° número: ";
		cin >> vetor[i];
	}

	for (int i = 0; i < (N - 1); i++) {
		aux = vetor[i];
		menor = vetor[i+1];
		pos = i + 1;
		for (int j = i + 1; j < N; j++) {
			if (vetor[j] < menor) {
				menor = vetor[j];
				pos = j;
			}
		}
		if (menor < aux) {
			vetor[i] = vetor[pos];
			vetor[pos] = aux;
		}
	}

	for (int i = 0; i < N; i++) {
		cout << (i + 1) << "° número: " << vetor[i] << "\n";
	}

	return 0;
}

File: python/selection_sort.py

N = 5
aux = -1
vetor = [None] * N

for i in range(N):
	vetor[i] = int(input('Entre com o ' + str(i + 1) + 'º valor: '))


for i in range(N - 1):
	aux = vetor[i]
	menor = vetor[i+1]
	pos = i + 1
	for j in range(i + 1, N):
		if vetor[j] < menor:
			menor = vetor[j]
			pos = j
	if menor < aux:
		vetor[i] = vetor[pos]
		vetor[pos] = aux

for i in range(N):
	print(str(i + 1) + '° número: ' + str(vetor[i]), end='\n')
